README.md

```diff
@@ -326,7 +326,7 @@ In the repository, make a virtual environment (`python -m venv .venv`) and enter
 Linux, or for Windows use one of `.venv\Scripts\activate.ps1`, `.venv\Scripts\activate.bat`,
 `source .venv/Scripts/activate`).
 
-The first thing you should run is `pip install -r dev-requirements.txt` to install nox.
+The first thing you should run is `pip install --group nox` to install nox.
```

Comment:
A note should probably made in the developing section here pointing users to install uv for a more streamlined dev experience. Something like

>[!TIP]
> We use the `uv` package manager as a faster drop-in replacement for pip. While it's not required to use in order to contribute to hikari, it's highly recommended! If you use it, replace `pip` commands below with `uv pip`.

Or mentioning `uv sync` instead if you choose to switch to that